Lab-6.ipynb

```diff
@@ -0,0 +1,631 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# plot the bkg\n",
+    "fig, ax = plt.subplots(1,1, figsize=(10,10))\n",
+    "plt.style.use('ggplot')\n",
+    "\n",
+    "ax.set_title(\"Poisson Background Distribution, $\\mu=100$\",fontsize=20)\n",
+    "ax.set_xlabel('Background Value',fontsize=20)\n",
+    "ax.set_ylabel('Probability',fontsize=20)\n",
+    "ax.tick_params(labelsize = 14)\n",
+    "\n",
+    "ax.hist(bkg,histtype='step',color='r',density=True,linewidth=3.5,bins=100)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To begin, we generate random Poisson-distributed values with a predefined mean of $\\mu=100$. This plot allows us to look at the shape of the background distribution before we begin injecting signals. We can see it follows the discrete-valued nature as expected for a Poisson, and is centered at the mean we  defined. Setting density to be true allows us to look at the probabilities instead of count frequencies for our background, but either verify that we created our fake background correctly."
```

Comment:
Good. The last sentence sounds funny. I believe you meant to write rather than either?